doc/active_docs/account_management_api.json

```diff
@@ -2383,15 +2383,13 @@
             "name": "id",
             "in": "path",
             "description": "ID of the user.",
-            "x-data-threescale-name": "user_ids",
```

Comment:
It's deleted here, and it's expected, because the endpoint says "change role to admin", but in this auto-complete all users are already admins.

Reply:
Correct me if I'm wrong: from the point of view of a provider, `user_ids` refers to users belonging to buyers, and `admin_ids` refers to users belonging to this provider account, but, only those with admin permissions? There can be some provide users that have access to the admin portal but they are not admins. Shouldn't those be autocompleted here?

Unrelated question: shouldn't all endpoints like `/admin/api/users/*` to be under the `Provider users` section?

Reply:
> from the point of view of a provider, `user_ids` refers to users belonging to buyers, and `admin_ids` refers to users belonging to this provider account, but, only those with admin permissions?

@jlledom Yes, that's correct

> There can be some provide users that have access to the admin portal but they are not admins. Shouldn't those be autocompleted here?

Yes, these are "member" admin users. That's what I tried to explain in my comment. This endpoint is for changing the user's role from `member` to `admin`, so I thought that it didn't make sense to list those who are already admins... 

> Unrelated question: shouldn't all endpoints like /admin/api/users/* to be under the Provider users section?

Yeah, I think it would be a nice improvement. I'll add it also.

Reply:
> Yes, these are "member" admin users. That's what I tried to explain in my comment. This endpoint is for changing the user's role from member to admin, so I thought that it didn't make sense to list those who are already admins...

But it would make sense to list members that can be promoted to admins.

Reply:
Well, yeah... it's just that there was no auto-complete list for this. But well, I can add it :grimacing:

Reply:
@jlledom I've added `provider_users_id` item for auto-complete.

So, it's used instead of `admin_ids`. I think it's not worth to split it into admins and members (i.e. have separate auto-complete for each of them)

Reply:
Cool